pkg/config/config.go

```diff
@@ -21,15 +21,21 @@ import (
 )
 
 type Config struct {
+	userAgent        string
 	defaultProjectID string
 }
 
+func (c *Config) UserAgent() string {
+	return c.userAgent
+}
+
 func (c *Config) DefaultProjectID() string {
 	return c.defaultProjectID
 }
 
 func New() *Config {
 	return &Config{
+		userAgent:        "gke-mcp",
```

Comment:
Can we include the version here as well? Something like "gke-mcp/0.0.1". You might need to plumb the version to the config as well.